lib/ejs.js

```diff
@@ -576,7 +576,8 @@ Template.prototype = {
 
     function _addOutput() {
       if (self.truncate) {
-        line = line.replace('\n', '');
+        line = line
+                  .replace(/\r?\n/, '')
```

Comment:
Ok, so it should match `\r` `\n` `\r\n` 

I don't see an easy way to put that in a regexp right now (because putting \r?\n? would also match anything,

Maybe /(\r\n|\r|\n)/ ?